src/components/ReminderCtaButton.tsx

```diff
@@ -0,0 +1,186 @@
+        display: flex;
+        flex-direction: column;
+        flex-wrap: wrap;
+        justify-content: flex-start;
+        align-items: flex-start;
+        max-width: 50%;
+    `,
+    buttonMargins: css`
+        margin: ${space[1]}px ${space[2]}px ${space[1]}px 0;
+    `,
+    button: css`
+        background-color: ${news[400]};
+        color: ${neutral[100]};
+        &:hover {
+            background-color: ${news[400]};
+        }
+    `,
+    thankYouText: css`
+        ${body.medium({ fontWeight: 'bold' })};
+        color: ${neutral[0]};
+        margin-top: 12px;
+    `,
+    errorText: css`
+        color: ${neutral[0]};
+        margin-top: 12px;
+    `,
+    remindMeButtonOverrides: css`
+        background-color: transparent !important;
+        color: ${neutral[7]} !important;
+
+        :hover {
+            background-color: ${neutral[86]} !important;
+        }
+    `,
+    smallPrint: css`
+        margin-top: 8px;
+    `,
+};
+
+interface RemindMeButtonProps {
+    disabled: boolean;
+    ctaText: string;
+    onClick: () => void;
+}
+
+const RemindMeButton = ({ disabled, ctaText, onClick }: RemindMeButtonProps) => (
+    <div css={styles.buttonMargins}>
+        <ThemeProvider theme={contributionsTheme}>
+            <Button
+                disabled={disabled}
+                onClick={() => onClick()}
+                priority="tertiary"
+                css={styles.remindMeButtonOverrides}
+            >
+                {ctaText}
+            </Button>
+        </ThemeProvider>
+    </div>
+);
+
+interface SmallPrintProps {
+    month: string;
+}
+
+const SmallPrint = ({ month }: SmallPrintProps) => (
+    <div css={styles.smallPrint}>
+        We will send you a maximum of two emails in {month}. To find out what personal data we
+        collect and how we use it, view our{' '}
+        <a href="https://manage.theguardian.com/email-prefs">Privacy Policy</a>.
+    </div>
+);
+
+interface ReminderCtaButtonProps {
+    reminderStage: ReminderStage;
+    reminderOption?: string;
+    internalButtonId: number;
+    ophanComponentId: string;
+    trackClick: TrackClick;
+    fetchEmail: FetchEmail;
+}
+
+export const ReminderCtaButton = ({
+    reminderStage,
+    reminderOption,
+    internalButtonId,
+    ophanComponentId,
+    trackClick,
+    fetchEmail,
+}: ReminderCtaButtonProps): JSX.Element => {
+    const { reminderCta, reminderPeriod, reminderLabel } = buildReminderFields();
+    const [remindState, setRemindState] = useState<ReminderStatus>('DEFAULT');
+
+    const onClick = () => {
+        trackClick({ internalButtonId, ophanComponentId });
+
+        setRemindState('IN_PROGRESS');
+
+        fetchEmail()
+            .then((email) => {
+                if (email) {
+                    return createReminder({
+                        reminderPeriod,
+                        email,
+                        reminderPlatform: 'WEB',
+                        reminderComponent: 'EPIC',
```

Comment:
I wonder if it'd be useful to be able to distinguish between the RRCP epic and Braze epic?

Reply:
Suggest it to TX tomorrow? But maybe not for this PR?